content/basics/use-ably.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: How to use Ably
+meta_description: "Determine how best to integrate Ably into your application with an overview of Ably's SDKs, APIs and protocol adapters."
+---
+
+There are several different ways to use Ably in an application. These include using an Ably SDK, choosing a protocol adapter for libraries such as MQTT or SSE, or utilizing an integration to communicate between Ably and external systems. Each method provides slightly different benefits and feature sets. This page discusses the options available.
+
+h2(#sdks). Ably SDKs with realtime and REST interfaces
+
+Ably SDKs provide a consistent and idiomatic API across a variety of "supported platforms":/getting-started/sdks#sdks and are the most feature-rich method of integrating Ably into an application. Ably SDKs contain a realtime and a REST interface, each of which can be used to satisfy different use cases.
+
+The REST interface communicates with the Ably service using the HTTP protocol and is effectively stateless. The REST interface provides a convenient way to access the "REST HTTP API":#rest-api and is intended to be used by clients that don't require realtime updates. It is more commonly used server-side. It is used to: 
+
+* publish messages on behalf of other clients
```

Comment:
Would it be worth noting that it can publish messages on behalf of itself as well?

Reply:
That's a very good point..! Updated.